packages/nx-playwright/src/generators/project/lib/add-linting.ts

```diff
@@ -27,10 +27,10 @@ export async function addLinting(
   });
 
   if (options.linter === Linter.EsLint) {
-    updateJson(
-      host,
-      joinPathFragments(options.projectRoot, '.eslintrc.json'),
-      (json: EslintIgnoreFile) => {
+    const eslintrcPath = joinPathFragments(options.projectRoot, '.eslintrc.json');
```

Comment:
The latest version of nx uses a `eslint.config.js` file by default rather than the `.eslintrc.json` file used in Onyx. This means that when this repo's pipeline is run, the nx next app generated does not contain an `.eslintrc.json` file and this step fails without checking for the file